x-pack/plugins/security_solution/scripts/endpoint/action_responder/utils.ts

```diff
@@ -95,6 +97,7 @@ export const sendEndpointActionResponse = async (
         command: action.command as EndpointActionData['command'],
         comment: '',
       },
+      ...getOutputDataIfNeeded(action.command as EndpointActionData['command']),
```

Comment:
The `output` data should not go into the endpoint index doc.

Reply:
Yeah, we need it in order to use it later when constructing the `ActionDetails`